### ISSUE TITLE
lattice_tune::lora::apply::adapt_step for online single-event gradient steps

### ISSUE THREAD

**Issue by ohdearquant** [ohdearquant/lattice#61]
khive's brain LoRA evolver (khive ADR-032 §5b) needs an *online* per-event gradient step over `LoraAdapter` weights — not the full-corpus batched training loop in `lattice_tune::train::TrainingLoop`.

Current state in `crates/tune/src/lora/apply.rs`: only `apply_lora` exists (inference, no mutation). `crates/tune/src/train/jit.rs::train_step_cpu` (line 368) is explicitly marked "Placeholder CPU training step" — returns a synthetic loss and touches no weights. lattice ADR-033 §Consequences/Negative confirms: "CPU training step is a placeholder…does not modify any weights" and "`JitStrategy::LowRank` defines the concept but the layer-level LoRA matrix management is not implemented."

Shape khive wants (any equivalent home in the crate is fine — likely a new module `crates/tune/src/lora/online.rs` rather than extending `apply.rs`, since `apply.rs` is scoped to immutable inference):

```rust
pub fn adapt_step(
    adapter:       &mut LoraAdapter,
    signal:        &FeedbackSignal,    // target embedding, current embedding, reward
    learning_rate: f32,
) -> AdaptStepResult;
```

Semantics: one SGD step (or its preferred online flavor — Adam/AdamW per-parameter state can be a follow-up). Mutates `adapter.layers[...].a` and `adapter.layers[...].b` in place. Deterministic given identical starting weights and signal. No async, no IO — same purity contract as `apply_lora`.

`FeedbackSignal` can be khive-owned (passed in as a `&[f32]` query embedding + `&[f32]` target embedding + reward scalar) — lattice doesn't need to take a dependency on khive's types.

Until this exists, khive implements its own SGD step in `khive-pack-brain::lora::sgd_step` against `LoraAdapter`'s public fields. When lattice ships `adapt_step`, khive delegates and removes the local impl. This also fills the gap that `train_step_cpu` flagged as a placeholder.

References:
- khive ADR-032 §5b (LoraEvolver::reduce)
- `crates/tune/src/lora/apply.rs`
- `crates/tune/src/train/jit.rs` line 368 (train_step_cpu placeholder)
- lattice ADR-033 §Consequences (negative)

**Comment by ohdearquant** [comment]
Already implemented. `adapt_step` ships at `crates/tune/src/lora/online.rs:38` for online single-event gradient steps, with 4 unit tests in the same module (all green under `cargo test -p lattice-tune --features train-backward`). Closing as done.
